lib/Client.js

```diff
@@ -953,11 +953,14 @@ class Client extends EventEmitter {
     * @arg {String} overwriteID The ID of the overwritten user or role (everyone role ID = guild ID)
     * @arg {BigInt} allow The permissions number for allowed permissions
     * @arg {BigInt} deny The permissions number for denied permissions
-    * @arg {String} type The object type of the overwrite, either "member" or "role"
+    * @arg {Number} type The object type of the overwrite, either 1 for "member" or 0 for "role"
     * @arg {String} [reason] The reason to be displayed in audit logs
     * @returns {Promise}
     */
     editChannelPermission(channelID, overwriteID, allow, deny, type, reason) {
+        if(isNaN(type)) { // backward compatbility
```

Comment:
Wouldn't it be better to check `typeof type === "string"` instead? This may have confusing or unintended behavior as is.